app/ashlar/README.md

```diff
@@ -0,0 +1,12 @@
+### Migrations
+
+Before creating a migration, read this:
+http://alembic.readthedocs.org/en/latest/autogenerate.html#what-does-autogenerate-detect-and-what-does-it-not-detect
```

Comment:
Ha yup. I had to modify this first migration to not attempt to delete the postgis spatial_ref_sys tables.